syntax = "proto3";
package streetgangpb;


message ErrorResponse
{
	string track_id = 1;
	int32 result = 2;
    int32 request_id = 3;
	string error_message = 4;
}
